distributed/client.py

```diff
@@ -3033,6 +3034,15 @@ def _get_computation_code(
                 or fr.f_code.co_name in ("<listcomp>", "<dictcomp>")
             ):
                 continue
+
+            # Ignore IPython specific wrapping IPython interactive related functions
+            elif (
+                hasattr(fr.f_back, "f_globals")
+                and sys.modules[fr.f_back.f_globals["__name__"]].__name__  # type: ignore
+                == "IPython.core.interactiveshell"
+            ):
+                if ipython_idx is None:
+                    ipython_idx = i - 2  # Everything past this index.
```

Comment:
The hardcoded `2` feels very magical. Is there a way to make this more descriptive? Also, is this prone to breaking in the future?

Reply:
`i` starts at 1, so it's minus 2 to get to the frame before this IPython frame (which would be the 0th index) Does that make sense? If so I'll update with a similar comment.